app/routes/__auth/login.ts

```diff
@@ -105,6 +106,7 @@ export const loader: LoaderFunction = async ({ request: { headers, url } }) => {
 
     const user = userFromTokenSet(tokenSet)
     const session = await sessionPromise
+    session.set('token', tokenSet.access_token)
```

Comment:
Might as well call this field `accessToken` or `access_token`, to distinguish it from the many other kinds of tokens (access, ID, refresh).

```suggestion
    session.set('accessToken', tokenSet.access_token)
```